File: poo/constructor/class.ts
class Employee {
  private name: string;
  private id: number;

  constructor(id: number, name: string) {
    this.id = id;
    this.name = name;
  }

  public getName(): string {
    return this.name;
  }

  public setName(name: string): void {
    this.name = name;
  }

  public getId(): number {
    return this.id;
  }

  public setId(id: number): void {
    this.id = id;
  }

}

const employee = new Employee(1, 'Name');

console.log(employee.getId());
console.log(employee.getName());

File: poo/constructor/class.py
class Employee():

  def __init__(self, id, name):
    self.id = id
    self.name = name

  def set_id(self, id):
    self.id = id

  def set_name(self, name):
    self.name = name

  def get_id(self):
    return self.id

  def get_name(self):
    return self.name


employee = Employee(1, 'Name')

print(employee.get_id())
print(employee.get_name())
